src/playlist-maker.js

```diff
@@ -22,144 +30,175 @@ var indexInSources = function(arr, src) {
   return -1;
 };
 
-// factory method to return a new playlist with the following API
```

Comment:
Actually, on second look, this format was replaced by the doc blocks for individual methods below. I'll verify that they're structured in a way that they can be output in meaningful API documentation.